assignments/prototypes.js

```diff
@@ -41,8 +41,55 @@
 
 // Test you work by un-commenting these 3 objects and the list of console logs below:
 
-/*
-  const mage = new Humanoid({
+function GameObject(gameObjectItem) {
+  this.createdAt = gameObjectItem.createdAt;
+  this.dimensions = gameObjectItem.dimensions;
+};
+
+GameObject.prototype.destroy = function () {
+  return `Object was removed from the game.`
+};
+
+function CharacterStats(statsItem) {
+  GameObject.call(this, statsItem);
+  this.healthPoints = statsItem.healthPoints;
+  this.name = statsItem.name;
+};
+
+CharacterStats.prototype.takeDamage = function () {
+  return `${this.name} took damage.`
+};
+
+CharacterStats.prototype = Object.create(GameObject.prototype);
+
+function CharacterStats(statsItem) {
+  GameObject.call(this, statsItem);
+}
+
+function Humanoid(humanoidItem) {
+  GameObject.call(this, humanoidItem);
+  CharacterStats.call(this, humanoidItem);
+  this.team = humanoidItem.team;
+  this.weapons = humanoidItem.weapons;
+  this.language = humanoidItem.language;
+};
+
```

Comment:
Just slip that in up here instead.

```suggestion
Humanoid.prototype = Object.create(CharacterStats.prototype);

```